packages/drivers/odsp-driver/src/odspDocumentServiceFactory.ts

```diff
@@ -35,3 +40,70 @@ export class OdspDocumentServiceFactory
         );
     }
 }
+
+export class LocalOdspDocumentServiceFactory extends OdspDocumentServiceFactoryCore {
+    constructor(
+        private readonly fluidFile: Uint8Array | string,
+    ) {
+        super(
+            async (_options) => { return null; }, // TODO
+            undefined,
+            async () => getSocketIo(), // TODO
+            undefined,
+            undefined, // TODO
+        );
+    }
+
+    public async createContainer(
+        _createNewSummary: ISummaryTree | undefined,
+        _createNewResolvedUrl: IResolvedUrl,
+        logger?: ITelemetryBaseLogger,
+        _clientIsSummarizer?: boolean,
+    ): Promise<IDocumentService> {
+        const odspLogger = createOdspLogger(logger);
+
+        return this.createDocumentServiceCore(
```

Comment:
I believe this API is called only when we are creating new instance of a container, which should never happen in this scenario. So, this call should just log & throw.